todo/src/reducers/index.js

```diff
@@ -0,0 +1,52 @@
+import {ADD} from '../actions'
+import {COMPLETED} from '../actions'
+
+
+
+const initialState = {
+    todos: [{
+        name: "take out the trash",
+        id: 0,
+        completed: false}],
+    input: "",
+}
+
+
+
+const reducer = (state = initialState, action) => {
+    switch(action.type) {
+        case ADD:
+        console.log("Added")
+        var newTask = {};
```

Comment:
use a const if the variable isn't changing, and a let if the variable is changing